src/locale/locales/ja/messages.po

```diff
@@ -46,10 +58,18 @@ msgstr "{0, plural, other {#分}}"
 msgid "{0, plural, one {# month} other {# months}}"
 msgstr "{0, plural, other {#ヶ月}}"
 
+#: src/screens/Post/PostQuotes.tsx:41
+msgid "{0, plural, one {# quote} other {# quotes}}"
+msgstr "{0, plural, other {# 引用}}"
+
 #: src/view/com/util/post-ctrls/RepostButton.tsx:69
 msgid "{0, plural, one {# repost} other {# reposts}}"
 msgstr "{0, plural, other {#回のリポスト}}"
 
+#: src/screens/Post/PostRepostedBy.tsx:41
+msgid "{0, plural, one {# reposts} other {# reposts}}"
```

Comment:
FYI #7064 just got merged to fix a typo here (thanks @auroursa!)

Reply:
thanks for your notification!!